recipes/protondrive/webview.js

```diff
@@ -0,0 +1,8 @@
+const _path = _interopRequireDefault(require('path'));
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+module.exports = (Ferdium) => {
```

Comment:
If the config setting in `package.json` has `hasIndirectMessages` as `true`, then there should be some corresponding code in this module to call `Ferdium.setBadge()`. Without such a call, the badges won't appear. Is that ok?

Reply:
Should be fine. I'm actually not sure if they ever send any browser notifications, never seen one. If this changes -- I'll update it accordingly.